package.json

```diff
@@ -17,44 +17,44 @@
   ],
   "repository": "eclipse-velocitas/cli",
   "dependencies": {
-    "@oclif/core": "^2.9.3",
-    "@oclif/plugin-help": "^5.2.13",
-    "@oclif/plugin-not-found": "^2.3.30",
-    "@oclif/plugin-plugins": "^3.1.6",
+    "@oclif/core": "^2.15.0",
```

Comment:
Can we use version without "^"  to create stable BL?

Reply:
no i would not do it for now, since we are not using any automated update or releasing

Reply:
We have a dependabot who notifies us about the version update. We need to create follow-up to make it automatic.
Right now, behavior is undefined for each commit - that technically means that the version that we release and the version that will be locally run on the machine - will be deviated.

Reply:
TODO: https://github.com/eclipse-velocitas/cli/issues/178